package.json

```diff
@@ -79,25 +81,26 @@
     "wait-4-docker": "node ./scripts/_wait4_all.js"
   },
   "dependencies": {
+    "@hapi/basic": "7.0.2",
+    "@hapi/catbox-memory": "6.0.2",
     "@hapi/good": "9.0.1",
     "@hapi/hapi": "21.3.10",
-    "@hapi/basic": "7.0.2",
     "@hapi/inert": "7.1.0",
     "@hapi/joi": "17.1.1",
     "@hapi/vision": "7.0.3",
-    "@hapi/catbox-memory": "6.0.2",
-    "@mojaloop/database-lib": "11.0.5",
     "@mojaloop/central-services-error-handling": "13.0.1",
     "@mojaloop/central-services-health": "15.0.0",
```

Comment:
is it missing dependency @mojaloop/sdk-standard-components required by @mojaloop/central-services-error-handling